utils/test/src/main/kotlin/Utils.kt

```diff
@@ -146,3 +147,7 @@ fun <T, U> transformingCollectionMatcher(
 fun <T, U> transformingCollectionEmptyMatcher(
     transform: (T) -> Collection<U>
 ): Matcher<T?> = neverNullMatcher { value -> beEmpty<U>().test(transform(value)) }
+
+fun ProjectAnalyzerResult.withInvariantIssues() =
+    // Account for different NPM versions to return issues in different order.
+    copy(issues = issues.sortedBy { it.message }.map { it.copy(timestamp = Instant.EPOCH) })
```

Comment:
As this is an extension function on an Analyzer-specific class, I'd propose to move it to `analyzer/src/testFixtures/kotlin/Extensions.kt` instead.